core/templates/components/button-directives/social-buttons.component.html

```diff
@@ -22,7 +24,7 @@
 
     <div class="oppia-twitter-follow">
       <a href="https://twitter.com/oppiaorg" target="_blank" rel="noopener">
-        <i class="oppia-follow-icons fab fa-twitter"></i>
+        <i class="oppia-follow-icons fa-brands fa-x-twitter"></i>
```

Comment:
Do we need this?

Reply:
Same here as well 😅

Reply:
I Guess I have done a bit mistake in committing new changes sorry from my side 😅